.github/workflows/nightly.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           restore-keys: ${{ runner.os }}-m2
 
       - name: Build with Maven
-        run: mvn clean package -Pcloud -DpostgreSqlJdbcUrl="jdbc:postgresql://postgres:5432/postgres" -DpostgreSqlUsername="${{ secrets.POSTGRES_USER }}" -DpostgreSqlPassword="${{ secrets.POSTGRES_PASSWORD }}" --file pom.xml
+        run: mvn clean package -Pcloud -Ddb.jdbcUrl="jdbc:postgresql://postgres:5432/postgres" -Ddb.user="${{secrets.POSTGRES_USER}}" -Ddb.password="${{secrets.POSTGRES_PASSWORD}}" --file pom.xml
```

Comment:
Please undo this change. It is unnecessary to the migration to Jakarta EE 10/Payara 6.